### PR TITLE
More and more WinForms enums

### PR DESCRIPTION
**Pull request by chenss3**
Fixes: #1092
Fixes: #1076
Fixes: #1121
Fixes: #1086
Fixes: #1089

Metadata Diff:
```
# DISPATCH_FLAGS
Windows.Win32.System.Com.IDispatch.Invoke : wFlags...UInt16 => DISPATCH_FLAGS
Windows.Win32.System.Com.ITypeInfo.Invoke : wFlags...UInt16 => DISPATCH_FLAGS
Windows.Win32.System.Ole.Apis.DISPATCH_METHOD removed
Windows.Win32.System.Ole.Apis.DISPATCH_PROPERTYGET removed
Windows.Win32.System.Ole.Apis.DISPATCH_PROPERTYPUT removed
Windows.Win32.System.Ole.Apis.DISPATCH_PROPERTYPUTREF removed
Windows.Win32.System.SystemServices.DISPATCH_FLAGS added
Windows.Win32.System.SystemServices.DISPATCH_FLAGS.DISPATCH_METHOD added
Windows.Win32.System.SystemServices.DISPATCH_FLAGS.DISPATCH_PROPERTYGET added
Windows.Win32.System.SystemServices.DISPATCH_FLAGS.DISPATCH_PROPERTYPUT added
Windows.Win32.System.SystemServices.DISPATCH_FLAGS.DISPATCH_PROPERTYPUTREF added
# VARFORMAT_FIRST_DAY
Windows.Win32.System.Ole.Apis.VarFormat : iFirstDay...Int32 => VARFORMAT_FIRST_DAY
Windows.Win32.System.Ole.Apis.VarTokenizeFormatString : iFirstDay...Int32 => VARFORMAT_FIRST_DAY
Windows.Win32.System.Ole.VARFORMAT_FIRST_DAY added
Windows.Win32.System.Ole.VARFORMAT_FIRST_DAY.VARFORMAT_FIRST_DAY_FRIDAY added
Windows.Win32.System.Ole.VARFORMAT_FIRST_DAY.VARFORMAT_FIRST_DAY_MONDAY added
Windows.Win32.System.Ole.VARFORMAT_FIRST_DAY.VARFORMAT_FIRST_DAY_SATURDAY added
Windows.Win32.System.Ole.VARFORMAT_FIRST_DAY.VARFORMAT_FIRST_DAY_SUNDAY added
Windows.Win32.System.Ole.VARFORMAT_FIRST_DAY.VARFORMAT_FIRST_DAY_SYSTEMDEFAULT added
Windows.Win32.System.Ole.VARFORMAT_FIRST_DAY.VARFORMAT_FIRST_DAY_THURSDAY added
Windows.Win32.System.Ole.VARFORMAT_FIRST_DAY.VARFORMAT_FIRST_DAY_TUESDAY added
Windows.Win32.System.Ole.VARFORMAT_FIRST_DAY.VARFORMAT_FIRST_DAY_WEDNESDAY added
# VARFORMAT_FIRST_WEEK
Windows.Win32.System.Ole.Apis.VarFormat : iFirstWeek...Int32 => VARFORMAT_FIRST_WEEK
Windows.Win32.System.Ole.Apis.VarTokenizeFormatString : iFirstWeek...Int32 => VARFORMAT_FIRST_WEEK
Windows.Win32.System.Ole.VARFORMAT_FIRST_WEEK added
Windows.Win32.System.Ole.VARFORMAT_FIRST_WEEK.VARFORMAT_FIRST_WEEK_CONTAINS_JANUARY_FIRST added
Windows.Win32.System.Ole.VARFORMAT_FIRST_WEEK.VARFORMAT_FIRST_WEEK_HAS_SEVEN_DAYS added
Windows.Win32.System.Ole.VARFORMAT_FIRST_WEEK.VARFORMAT_FIRST_WEEK_LARGER_HALF_IN_CURRENT_YEAR added
Windows.Win32.System.Ole.VARFORMAT_FIRST_WEEK.VARFORMAT_FIRST_WEEK_SYSTEMDEFAULT added
# VARFORMAT_NAMED_FORMAT
Windows.Win32.System.Ole.Apis.VarFormatDateTime : iNamedFormat...Int32 => VARFORMAT_NAMED_FORMAT
Windows.Win32.System.Ole.VARFORMAT_NAMED_FORMAT added
Windows.Win32.System.Ole.VARFORMAT_NAMED_FORMAT.VARFORMAT_NAMED_FORMAT_GENERALDATE added
Windows.Win32.System.Ole.VARFORMAT_NAMED_FORMAT.VARFORMAT_NAMED_FORMAT_LONGDATE added
Windows.Win32.System.Ole.VARFORMAT_NAMED_FORMAT.VARFORMAT_NAMED_FORMAT_LONGTIME added
Windows.Win32.System.Ole.VARFORMAT_NAMED_FORMAT.VARFORMAT_NAMED_FORMAT_SHORTDATE added
Windows.Win32.System.Ole.VARFORMAT_NAMED_FORMAT.VARFORMAT_NAMED_FORMAT_SHORTTIME added
# VARFORMAT_GROUP
Windows.Win32.System.Ole.Apis.VarFormatNumber : iGroup...Int32 => VARFORMAT_GROUP
Windows.Win32.System.Ole.Apis.VarFormatPercent : iGroup...Int32 => VARFORMAT_GROUP
Windows.Win32.System.Ole.VARFORMAT_GROUP added
Windows.Win32.System.Ole.VARFORMAT_GROUP.VARFORMAT_GROUP_NOTTHOUSANDS added
Windows.Win32.System.Ole.VARFORMAT_GROUP.VARFORMAT_GROUP_SYSTEMDEFAULT added
Windows.Win32.System.Ole.VARFORMAT_GROUP.VARFORMAT_GROUP_THOUSANDS added
# VARFORMAT_LEADING_DIGIT
Windows.Win32.System.Ole.Apis.VarFormatNumber : iIncLead...Int32 => VARFORMAT_LEADING_DIGIT
Windows.Win32.System.Ole.Apis.VarFormatPercent : iIncLead...Int32 => VARFORMAT_LEADING_DIGIT
Windows.Win32.System.Ole.VARFORMAT_LEADING_DIGIT added
Windows.Win32.System.Ole.VARFORMAT_LEADING_DIGIT.VARFORMAT_LEADING_DIGIT_INCLUDED added
Windows.Win32.System.Ole.VARFORMAT_LEADING_DIGIT.VARFORMAT_LEADING_DIGIT_NOTINCLUDED added
Windows.Win32.System.Ole.VARFORMAT_LEADING_DIGIT.VARFORMAT_LEADING_DIGIT_SYSTEMDEFAULT added
# VARFORMAT_PARENTHESES
Windows.Win32.System.Ole.Apis.VarFormatNumber : iUseParens...Int32 => VARFORMAT_PARENTHESES
Windows.Win32.System.Ole.Apis.VarFormatPercent : iUseParens...Int32 => VARFORMAT_PARENTHESES
Windows.Win32.System.Ole.VARFORMAT_PARENTHESES added
Windows.Win32.System.Ole.VARFORMAT_PARENTHESES.VARFORMAT_PARENTHESES_NOTUSED added
Windows.Win32.System.Ole.VARFORMAT_PARENTHESES.VARFORMAT_PARENTHESES_SYSTEMDEFAULT added
Windows.Win32.System.Ole.VARFORMAT_PARENTHESES.VARFORMAT_PARENTHESES_USED added
# GPFIDL_FLAGS
Windows.Win32.UI.Shell.Apis.GPFIDL_ALTNAME removed
Windows.Win32.UI.Shell.Apis.GPFIDL_DEFAULT removed
Windows.Win32.UI.Shell.Apis.GPFIDL_UNCPRINTER removed
Windows.Win32.UI.Shell.Apis.SHGetPathFromIDListEx : uOpts...Int32 => GPFIDL_FLAGS
Windows.Win32.UI.Shell.GPFIDL_FLAGS added
Windows.Win32.UI.Shell.GPFIDL_FLAGS.GPFIDL_ALTNAME added
Windows.Win32.UI.Shell.GPFIDL_FLAGS.GPFIDL_DEFAULT added
Windows.Win32.UI.Shell.GPFIDL_FLAGS.GPFIDL_UNCPRINTER added
# SCROLLBAR_COMMAND
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_BOTTOM removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_ENDSCROLL removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_LEFT removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_LINEDOWN removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_LINELEFT removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_LINERIGHT removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_LINEUP removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_PAGEDOWN removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_PAGELEFT removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_PAGERIGHT removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_PAGEUP removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_RIGHT removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_THUMBPOSITION removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_THUMBTRACK removed
Windows.Win32.UI.WindowsAndMessaging.Apis.SB_TOP removed
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_BOTTOM added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_ENDSCROLL added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_LEFT added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_LINEDOWN added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_LINELEFT added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_LINERIGHT added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_LINEUP added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_PAGEDOWN added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_PAGELEFT added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_PAGERIGHT added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_PAGEUP added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_RIGHT added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_THUMBPOSITION added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_THUMBTRACK added
Windows.Win32.UI.WindowsAndMessaging.SCROLLBAR_COMMAND.SB_TOP added
# OleCtl.h functions 
Windows.Win32.System.Ole.Apis.OleCreateFontIndirect : lpFontDesc : [In,Out] => [In]
Windows.Win32.System.Ole.Apis.OleCreateFontIndirect : lplpvObj : [In,Out] => [Out]
Windows.Win32.System.Ole.Apis.OleCreatePictureIndirect : lplpvObj : [In,Out] => [Out]
Windows.Win32.System.Ole.Apis.OleCreatePictureIndirect : lpPictDesc : [In,Out] => [In]
Windows.Win32.System.Ole.Apis.OleCreatePropertyFrame : pPageClsID : [In,Out] => [In]
Windows.Win32.System.Ole.Apis.OleCreatePropertyFrame : ppUnk : [Out] => [In]
Windows.Win32.System.Ole.Apis.OleCreatePropertyFrame : pvReserved : [In,Out] => [In]
Windows.Win32.System.Ole.Apis.OleCreatePropertyFrameIndirect : lpParams : [In,Out] => [In]
Windows.Win32.System.Ole.Apis.OleLoadPicture : lplpvObj : [In,Out] => [Out]
Windows.Win32.System.Ole.Apis.OleLoadPictureEx : lplpvObj : [In,Out] => [Out]
Windows.Win32.System.Ole.Apis.OleLoadPicturePath : ppvRet : [In,Out] => [Out]
Windows.Win32.System.Ole.Apis.OleTranslateColor : lpcolorref : [In,Out] => [Out]
```

### CODE REVIEW

File: generation/WinSDK/enums.json [hunk 1]
```diff
@@ -31894,6 +31894,298 @@
...
+    "flags": true,
+    "name": "DISPATCH_FLAGS",
+    "autoPopulate": {
+      "filter": "DISPATCH_",
+      "header": "OleAuto.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "IDispatch::Invoke",
```
Comment (mikebattista) [inline generation/WinSDK/enums.json:31909]: Didn't know you could do this. Looks like it works, but typically we define "interface" and "method" separately for these scenarios. Would recommend doing that for consistency.

File: generation/WinSDK/enums.json [hunk 2]
```diff
@@ -31894,6 +31894,298 @@
...
+        "parameter": "wFlags"
+      },
+      {
+        "method": "ITypeInfo::Invoke",
```
Comment (mikebattista) [inline generation/WinSDK/enums.json:31913]: Same as above.

**Review by mikebattista — CHANGES_REQUESTED** [review]

**Comment by elachlan** [comment]
@chenss3 What is the merge conflict on? Apparently I cannot see what is causing it.

**Comment by mikebattista** [comment]
> @chenss3 What is the merge conflict on? Apparently I cannot see what is causing it.

This branch doesn't contain the .gitattributes change. Sync with main and see if that helps. The conflict is in the changes file.

**Comment by chenss3** [comment]
> > @chenss3 What is the merge conflict on? Apparently I cannot see what is causing it.
> 
> This branch doesn't contain the .gitattributes change. Sync with main and see if that helps. The conflict is in the changes file.

Should be fixed now!

**Comment by elachlan** [comment]
When you synced with main, did the changes file merge fine?

**Comment by chenss3** [comment]
> When you synced with main, did the changes file merge fine?

When I synced with main that time, I still had to manually resolve the merge conflicts in the changes file.

**Review by mikebattista — DISMISSED** [review]

File: scripts/ChangesSinceLastRelease.txt [hunk 3]
```diff
@@ -414,6 +414,121 @@ Windows.Win32.UI.Controls.RichEdit.CHARFORMATA.bPitchAndFamily...Windows.Win32.G
...
+Windows.Win32.System.Ole.Apis.OleCreatePropertyFrame : pPageClsID : [In,Out] => [In]
+Windows.Win32.System.Ole.Apis.OleCreatePropertyFrame : ppUnk : [Out] => [In]
+Windows.Win32.System.Ole.Apis.OleCreatePropertyFrame : pvReserved : [In,Out] => [In]
+Windows.Win32.System.Ole.Apis.OleCreatePropertyFrameIndirect : lpParams : [In,Out] => [In]
+Windows.Win32.System.Ole.Apis.OleLoadPicture : lplpvObj : [In,Out] => [Out]
+Windows.Win32.System.Ole.Apis.OleLoadPictureEx : lplpvObj : [In,Out] => [Out]
+Windows.Win32.System.Ole.Apis.OleLoadPicturePath : ppvRet : [In,Out] => [Out]
+Windows.Win32.System.Ole.Apis.OleTranslateColor : lpcolorref : [In,Out] => [Out]
 # UIA_ enums
 Windows.Win32.System.SystemServices.UIA_ANNOTATIONTYPE added
```
Comment (mikebattista) [inline scripts/ChangesSinceLastRelease.txt:533]: APIs in the SystemServices namespace indicates a bug. Even though the UIA changes went in with another PR, please update enums.json to move anything in SystemServices over to a more relevant namespace like Accessibility.

Either add a usage so the enum follows the namespace of the APIs, or add an explicit namespace definition.

**Review by mikebattista — CHANGES_REQUESTED** [review]

**Pull request merged**